conda_smithy/cli.py

```diff
@@ -76,10 +77,10 @@ def generate_feedstock_content(target_directory, source_recipe_dir):
 
 class Subcommand:
     #: The name of the subcommand
-    subcommand = None
-    aliases = []
+    subcommand: Optional[str] = None
+    aliases: list = []
```

Comment:
Do we know the type of the list here so it can be `List`?

Reply:
added `List[str]`